src/helpers.js

```diff
@@ -6,6 +6,8 @@
    * @see https://stackoverflow.com/a/42604801/4422345 for where this code was taken from.
    */
 export const stringifyQuery = (params = {}, prefix) => {
+  if (! params) return '';
```

Comment:
When we add ESLint to this it will yell at you for not using `{...}`. Use them ;)